draft-ietf-httpbis-retrofit.md

```diff
@@ -189,86 +208,91 @@ Date: Sun, 06 Nov 1994 08:49:37 GMT
+For example, this Location field:
 
 ~~~ http-message
 Location: https://example.com/foo
 ~~~
 
-could be mapped as:
+would have a mapped value of:
 
-~~~ http-message
-SF-Location: "https://example.com/foo"
+~~~
+"https://example.com/foo"
 ~~~
 
 
 ## Dates
 
-The field names in {{date-fields}} (paired with their mapped field names) have values that can be mapped into Structured Fields by parsing their payload according to {{Section 5.6.7 of HTTP}} and representing the result as a Date.
+The field names in {{date-fields}} have values that can be mapped into Structured Field values by parsing their payload according to {{Section 5.6.7 of HTTP}} and representing the result as a Date.
 
-| Field Name          | Mapped Field Name      |
-|---------------------|------------------------|
-| Date                | SF-Date                |
-| Expires             | SF-Expires             |
-| If-Modified-Since   | SF-If-Modified-Since   |
-| If-Unmodified-Since | SF-If-Unmodified-Since |
-| Last-Modified       | SF-Last-Modified       |
+| Field Name          |
+|---------------------|
+| Date                |
+| Expires             |
+| If-Modified-Since   |
+| If-Unmodified-Since |
+| Last-Modified       |
 {:id="date-fields" title="Date Fields"}
 
-For example, an Expires field could be mapped as:
+For example, an Expires field's value could be mapped as:
 
 ~~~ http-message-new
-SF-Expires: @1659578233
+@1659578233
 ~~~
 
 ## ETags
 
-The field value of the ETag header field can be mapped into the SF-ETag Structured Field by representing the entity-tag as a String, and the weakness flag as a Boolean "w" parameter on it, where true indicates that the entity-tag is weak; if 0 or unset, the entity-tag is strong.
+The field value of the ETag header field can be mapped into a Structured Field value by representing the entity-tag as a String, and the weakness flag as a Boolean "w" parameter on it, where true indicates that the entity-tag is weak; if 0 or unset, the entity-tag is strong.
 
-For example, this:
+For example, this ETag header field:
 
 ~~~ http-message
 ETag: W/"abcdef"
 ~~~
 
-~~~ http-message
-SF-ETag: "abcdef"; w
-~~~
+would have a mapped value of:
 
-If-None-Match's field value can be mapped into the SF-If-None-Match Structured Field, which is a List of the structure described above. When a field value contains "*", it is represented as a Token.
+~~~
+"abcdef"; w
+~~~
 
-Likewise, If-Match's field value can be mapped into the SF-If-Match Structured Field in the same manner.
+If-None-Match's field value can be mapped into a Structured Field value which is a List of the structure described above. When a field value contains "*", it is represented as a Token.
 
+Likewise, If-Match's field value can be mapped into a Structured Field value in the same manner.
 
-For example:
+For example, this If-None-Match field:
 
 ~~~ http-message
-SF-If-None-Match: "abcdef"; w, "ghijkl", *
+If-None-Match: "abcdef"; w, "ghijkl", *
```

Comment:
This isn't valid.  Both because this uses a "; w" parameter instead of the leading "W/", and because it is not valid (also, nonsensical) to include "*" in addition to tags.

Reply:
Fixed.